pkg/plugins/autodiscovery/fleet/test/main_test.go

```diff
@@ -0,0 +1,89 @@
+package fleet
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/updatecli/updatecli/pkg/plugins/autodiscovery/fleet"
+)
+
+func TestDiscoverManifests(t *testing.T) {
+	// Disable condition testing with running short test
+	if testing.Short() {
+		return
+	}
+
+	testdata := []struct {
+		name              string
+		rootDir           string
+		expectedPipelines []string
+	}{
+		{
+			name:    "Scenario 1",
+			rootDir: "testdata",
+			expectedPipelines: []string{`name: 'Bump Fleet Bundle "grafana" for Helm Chart "grafana"'
```

Comment:
```suggestion
			expectedPipelines: []string{`name: 'Bump "grafana" Fleet bundle for "grafana" Helm chart'
```
Is a fleet included in a chart or the inverse?